lego/apps/comments/models.py

```diff
@@ -27,5 +27,14 @@ class Meta:
         ordering = ("created_at",)
         permission_handler = CommentPermissionHandler()
 
+    def delete(self):
+        with transaction.atomic():
+            self.created_by = None
+            self.text = "<p>Slettet</p>"
+            if Comment.objects.filter(parent=self).exists():
+                self.save()
+            else:
+                super(Comment, self).delete()
```

Comment:
Does it make more sense to do `delete(force=True)`. I don't see why this should be a `PersistentModel` at all

Reply:
Also, why set `created_by` and `text` before deleting (when using force)? Isn't it more logical to do that separatly? And, if soft deleting, does `.delete()` save the new values?